src/MongoDB.Analyzer/Core/Builders/BuilderExpressionProcessor.cs

```diff
@@ -483,6 +483,11 @@ private static SyntaxNode GetEnumCastNode(ITypeSymbol typeSymbol, object constan
             return RewriteResult.Ignore;
         }
 
+        if(SyntaxFactoryUtilities.IsMemberOfAnonymousObject(simpleNameSyntax))
```

Comment:
space between if and open bracket: `if (`

Reply:
Resolved